html/images/os/terra.svg

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
```

Comment:
SVG size can be cut down by 1/3, you can trim it down with: https://jakearchibald.github.io/svgomg/